js/widgets.js

```diff
@@ -272,6 +278,7 @@ function insertSearchInput (event, territoryElement) {
   searchNode.style.margin = '1em'
   searchNode.style.padding = '.1em'
   territoryElement.parentNode.insertBefore(searchNode, territoryElement)
+  // TODO: add a timeout between keyups.
```

Comment:
Adding `lodash.debounce` adds 5Kb to my build vs. 10 lines of unbloated ES6 that I understand. Not worth it IMHO.